doc/multiBand.dox

```diff
@@ -0,0 +1,73 @@
+  each source comes from. This produces a cross-band master source catalog. During the merging process, we 
+  cull garbage peaks around bright sources using the criteria described in \ref CullPeaksConfig 
+  "CullPeaksConfig".
+  \par Inputs:
+    deepCoadd_det{tract,patch,filter}: SourceCatalog (only parent Footprints)
+  \par Outputs:
+    deepCoadd_mergeDet{tract,patch}: SourceCatalog (only parent Footprints)
+  \par Data Unit: 
+    tract, patch</DD>
+  <DT> iii. \ref MeasureMergedCoaddSourcesTask_ "MeasureMergedCoaddSourcesTask": De-Blend & Measure Sources </DT> 
+  <DD> The master catalog is used to de-blend & measure sources in each coadd. This ensures that all bands 
+  will have the same sources. \ref SourceDeblendTask_ "SourceDeblendTask" is used to deblend sources. The 
+  deblender has to retain all peaks from the master catalog, deblending peaks that do not correspond to 
+  sources in the given coadd (dropouts) as PSFs. The \ref SingleFrameMeasurementTask_ "SingleFrameMeasurementTask" 
+  is used to measure the properties of the de-blended children. \ref SetPrimaryFlagsTask_ "SetPrimaryFlagsTask" 
+  is run to set 'is-primary' and related flags and \ref PropagateVisitFlagsTask_ "PropagateVisitFlagsTask" is 
+  used to propagte flags from individual visits to the coadd (eg. was the source used for PSF determination 
+  in the original visit etc...). At this point, the \ref ANetAstrometryTask_ "ANetAstrometryTask" can be run 
+  (optional, not run by default) to match the sources to a reference catalog. 
+  \par Inputs:
+    deepCoadd_mergeDet{tract,patch}: SourceCatalog
+    \n deepCoadd_calexp_det{tract,patch,filter}: ExposureF
+  \par Outputs:
+    deepCoadd_meas{tract,patch,filter}: SourceCatalog
+  \par Data Unit: 
+    tract, patch, filter</DD>
+  <DT> iv. \ref MergeMeasurementsTask_ "MergeMeasurementsTask" Merge Be-Blended Peaks</DT> 
+  <DD> De-blended peaks from each band are merged. We maintain track of a 'reference' band for each peak. 
+  This reference band is the first band from a priority-sorted list of bands (e.g. ["HSC-I", "HSC-R", "HSC-Z", 
+  "HSC-Y", "HSC-G",] in the case of HSC). Every source must have a centroid, shape, and CModel fit in every 
+  band, even if the object was not detected in the canonical band.
+  \par Inputs:
+    deepCoadd_meas{tract,patch,filter}: SourceCatalog
+  \par Outputs:
+    deepCoadd_ref{tract,patch}: SourceCatalog
+  \par Data Unit: 
+    tract, patch</DD>
+  <DT> v. Run Forced Photometry on Coadds</DT>
+  <DD> Forced photometry is run on the coadd for each band using reference centroids and shapes but using 
+  footprints from the previous de-blending step i.e. the footprints are band specific.</DD>
+  <DT> vi. Run Forced Photometry on Exposures</DT>
+  <DD> Forced photometry is run on the individual calibrated exposures for each band for each visit using the 
+  reference measurements (but no deblending).</DD>
```

Comment:
The measurement on exposures holds exactly the same parameters fixed as on coadds, doesn't it? (Do correct me if not). If so, can you use the same wording for the above two entries to make that clear.